src/node_api.h

```diff
@@ -127,9 +127,18 @@ NAPI_EXTERN napi_status napi_create_array(napi_env env, napi_value* result);
 NAPI_EXTERN napi_status napi_create_array_with_length(napi_env env,
                                                       size_t length,
                                                       napi_value* result);
-NAPI_EXTERN napi_status napi_create_number(napi_env env,
+NAPI_EXTERN napi_status napi_create_double(napi_env env,
```

Comment:
I think we should keep supporting the name `napi_create_number` to avoid the – minor – breakage that this creates.

That can be as simple as

```c++
#define napi_create_number napi_create_double
```

in the header and 

```c++
#undef napi_create_number
napi_status napi_create_number(napi_env env, double value, napi_value* result) {
  return napi_create_double(env, value, result);
}
```

in the source file.